app/Filament/Pages/Settings.php

```diff
@@ -542,8 +618,6 @@ public function save(): void
 
             $this->rememberData();
 
-            $this->redirect($this->getUrl());
```

Comment:
I dont see the need for this redirect.

Reply:
I think it was added to update the values on the settings page after you saved them? I could be wrong. @Boy132

Reply:
Yes, that's intentional. It's to refresh the page.

Reply:
But why? For what i could see the page does not need to refresh and it is rather annoying, because it does not redirect to the tab that you are in, but to the general tab.

Reply:
It depends on the settings you change. The refresh is to instantly show the changes.